skimage/filters/_sparse.py

```diff
@@ -16,9 +16,9 @@ def _validate_window_size(axis_sizes):
     """
     for axis_size in axis_sizes:
         if axis_size % 2 == 0:
-            msg = ('Window size for `threshold_sauvola` or '
-                   '`threshold_niblack` must not be even on any dimension. '
-                   'Got {}'.format(axis_sizes))
+            msg = (f'Window size for `threshold_sauvola` or '
+                   f'`threshold_niblack` must not be even on any dimension. '
+                   f'Got {axis_sizes}')
```

Comment:
The first two lines here don't have to have the `f` since there is no substitution there, but I don't think it hurts. I don't know if it is more common to leave it off in that case as below? Maybe @mkcor has an opinion? 

```Python
            msg = ('Window size for `threshold_sauvola` or '
                   '`threshold_niblack` must not be even on any dimension. '
                   f'Got {axis_sizes}')
```

Reply:
@grlee77 Yup, that should be fine as well. As far as I can tell, I don't think that there's too much out there in terms of style guidelines for this - I had left it this way after reading the PEP and a few posts like [this one](https://stackoverflow.com/questions/48471086/style-for-continued-multi-line-f-strings), as it felt like more readable to me and less likely to lead to wrong messages if they were changed in the future.

I don't feel strongly about it though and am happy to change it if you'd prefer the strings without placeholders to not be f-strings

Reply:
Good question! At first I thought: Why bother with an extra, unnecessary `f` prefix? I like to follow a minimalist approach. But after reading the SO post @alex-jw-brooks linked to, I'm happy to go for the multi-line `f` prefix!